plugin/restore/mysql/mysql.go

```diff
@@ -77,12 +81,88 @@ func (r *Restore) RestorePITR(ctx context.Context, fullBackup *bufio.Scanner, bi
 		return err
 	}
 
-	// TODO(dragonly): implement RestoreBinlog in mysql driver
-	_ = r.RestoreBinlog(ctx, binlog)
+	_ = r.replayBinlog(ctx, binlog)
```

Comment:
handle the error?

Reply:
Added TODO.
Should be handled when `replayBinlog` is implemented.